content/02/02.05/script.js.adoc

```diff
@@ -1,3 +1,187 @@
 include::./title.adoc[]
 
-TODO
+shot::[1]
+Once we have the boilerplate in place, we can start working on the actual code for our scenario.
```

Comment:
```suggestion
 Now that we have the step definitions matching, we can start working on our solution. We like to use our scenarios to guide our development, so we’ll start designing the objects we’ll need by sketching out some code in our step definitions.

The scenario will be failing while we do this, but we should see the error messages gradually progressing as we drive out the interface to our object model.

Our next goal is for the scenario to fail because we need to implement the actual business logic. Then we can work on changing the business logic inside our objects to make it pass.

```